tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+# role tasks
+- name: Upgrade kexec-tools to latest
+  yum:
+    name: kexec-tools
+    state: latest
+  become: true
+  become_user: "{{ kdump_become_user }}"
+
+- name: Waive previous nfs setup if enabled
+  lineinfile: dest="{{ kdump_conf_path }}"
+              state=present
+              regexp='^nfs (.*)$'
+              line='#nfs \1'
+              backup=true
+              backrefs=true
+  become: true
+  become_user: "{{ kdump_become_user }}"
+
+- name: Set nfs kdump target for crash kernel
+  lineinfile: dest="{{ kdump_conf_path }}"
+              state=present
+              regexp='^(#?)dracut_args (.*)$'
+              line='dracut_args --mount "kdump.usersys.redhat.com:/var/www/html/vmcore /mnt/vmcore nfs defaults"'
+              backup=true
+  become: true
+  become_user: "{{ kdump_become_user }}"
+
+- name: Set relative path of vmcore folder
+  lineinfile: dest="{{ kdump_conf_path }}"
```

Comment:
How large is the kdump.conf file? At a certain size it becomes preferable to run these lineinfile commands as a single `template` update. Alternatively, can we look at combining all three of these into a single call using a loop?

Reply:
The file is a little over 160 lines long